### PR TITLE
built a function to measure query times.

### PR DESCRIPTION
**Pull request by AngieD8615**
Brought query time down to less than 0.25msec for styles and 0.15msec for prod

### CODE REVIEW

File: DataBase/mongoQueryTimes.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+use products
```
Comment (elocowan) [inline DataBase/mongoQueryTimes.js:1]: curious what these lines are for

File: DataBase/mongoQueryTimes.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+var startStat = new Date();
+randIndexes.forEach((el, idx) => {
+  randStyleStat[idx] = db.styles.find({ product_id: el }).explain("executionStats").executionStats.executionTimeMillis;
+})
+var endStat = new Date();
+// print("index", randIndexes);
+print(`time to complete ${numbOfQueries} Styles queries:`, endStat - startStat, `Ave time: (end - start) / ${numbOfQueries}`, (endStat - startStat) / numbOfQueries);
+
+var prodBar = randProdStat.reduce((acc, cur) => acc + cur) / numbOfQueries;
+var styleBar = randStyleStat.reduce((acc, cur) => acc + cur) / numbOfQueries;
```
Comment (elocowan) [inline DataBase/mongoQueryTimes.js:27]: I think this is great work, and I'll be interested to hear whether new-relic makes some of this redundant or not.

**Review by elocowan — APPROVED** [review]
Hey Angie - this looks great. The only question I have is about the first two lines of mongoQueryTimes.js, which seem like they would throw errors, but I must just be missing something. AND, I'll be curious to hear if New Relic covers some of the same functionality that you made for yourself with that queryTimes files.

File: DataBase/mongoQueryTimes.js (hunk 1 above)
Comment (AngieD8615) [reply DataBase/mongoQueryTimes.js:1]: good question! this file is to be called within the terminal in conjuncture with opening mongo.  The first line instructs mongo to use the correct database and the second has mongo display the available collections within the products db inside mongo

**Pull request merged**